notification/slack/util.go

```diff
@@ -0,0 +1,36 @@
+package slack
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/target/goalert/config"
+)
+
+type Meta struct {
+	TeamID string `json:"team_id"`
+}
```

Comment:
This should probably just go inside the `getTeamID` func for parsing the return value